code/modules/organs/organ_external.dm

```diff
@@ -1127,7 +1127,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 
 //Is the limb robotic and malfunctioning
 /datum/organ/external/proc/is_malfunctioning()
-	return (is_robotic() && ((status & ORGAN_MALFUNCTIONING) || prob(brute_dam + burn_dam)))
+	return (return (is_robotic() && ((status & ORGAN_MALFUNCTIONING) || (brute_dam + burn dam > 14 && prob(brute_dam + burn_dam)))))
```

Comment:
change it to
`	return (is_robotic() && (status & (ORGAN_MALFUNCTIONING) || (brute_dam + burn_dam > min_broken_damage/2 && prob(brute_dam + burn_dam))))`

That'll work, and unsnowflake that 14 to something we can work with